e2e/testsuite/grpc_query.go

```diff
@@ -0,0 +1,23 @@
+package testsuite
+
+import (
+	"context"
+
+	"github.com/strangelove-ventures/ibctest/ibc"
+
+	channeltypes "github.com/cosmos/ibc-go/v5/modules/core/04-channel/types"
+)
+
+// QueryPacketCommitment queries the packet commitment on the given chain for the provided channel and sequence.
```

Comment:
Maybe we should add a AssertPacketIsRelayed function?

Reply:
Sounds like a useful assertion to have, we can always add it as we need it though if you don't want to include it in this PR.